### ISSUE TITLE
Package for Distributions

### ISSUE THREAD

**Issue by ZaneA** [ZaneA/WacomWebPlugin#8]
As the plugin stabilises, I would love to package it for both ArchLinux and Ubuntu/Debian.

ArchLinux can be handled easily via AUR, and I would like to handle Ubuntu via a PPA, but plain .deb's should be easy to create too.

**Comment by ZaneA** [comment]
ArchLinux now has a PKGBUILD at https://aur.archlinux.org/packages/wacomwebplugin/

**Comment by wavewave** [comment]
I would like to ask whether it is possible to package this plugin for firefox extension for android. Samsung galaxy note series has wacom, so this will enable to make a killer web app with full wacom feature. :-D

**Comment by wavewave** [comment]
forget about my previous comment. I forgot that android does not use X11..

**Comment by ZaneA** [comment]
True, but a great suggestion regardless. Once the Wayland display server (http://wayland.freedesktop.org/) is common-place I plan on transitioning the plugin to use that instead of X11 + XInput. I'm not sure how input works on Wayland currently but it might get the project closer to being able to support Android (assuming Samsung is reusing regular Linux Wacom drivers). When that rewrite happens I'll make sure to abstract the input out if needed to support Android.

The only problem is that I don't have a timeline for this! For now I can only say it will be "as soon as Wayland is workable on my desktop", hopefully not too long now :)

**Comment by ZaneA** [comment]
Ah it looks like Wayland and Android both (from what I can tell) use `evdev` for input. I will investigate further as I might be able to use that directly with X11 too.

**Comment by wavewave** [comment]
The following link is clearly helpful for developing android support.
http://www.slideshare.net/TNaruto/how-to-add-nfc-web-ap-is-to-android-by-using-npapi

**Comment by ZaneA** [comment]
Thanks! Will check that out.
